package.json

```diff
@@ -14,10 +14,18 @@
         "postdownload-api": "vscode-dts main",
         "webpack": "webpack --mode production && node ./scripts/optimizeTypings.js",
         "webpack-link": "webpack --mode development && node ./scripts/optimizeTypings.js",
-        "webpack-dev": "webpack --mode development --watch"
+        "webpack-dev": "webpack --mode development --watch",
+        "postinstall": "npm run download-api"
     },
     "author": "Visual Studio Code Team",
     "license": "MIT",
+    "engines": {
+        "vscode": "^1.64.0"
+    },
+    "enabledApiProposals": [
```

Comment:
Are these necessary for something that doesn't create an extension?

Reply:
at the end of the day we are still using the propsed api indirectly, we don't need to have crafty the dts.
ie the dts files area manually created, with this approach we don't need to do that anymore, uts pulled down automatically

Reply:
Forgot this dictates what .d.ts files are pulled down.